lib/ui/components/toggle_switch_widget.dart

```diff
@@ -60,7 +60,7 @@ class _ToggleSwitchWidgetState extends State<ToggleSwitchWidget> {
               activeColor: enteColorScheme.primary400,
               inactiveTrackColor: enteColorScheme.fillMuted,
               materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
-              value: toggleValue,
+              value: toggleValue ?? false,
```

Comment:
`toggleValue` is non-nullable, why is this needed?

Reply:
I was seeing null values in errors. Does it depend on some package/method which is not null safe?